src/Illuminate/Bus/BusServiceProvider.php

```diff
@@ -26,11 +28,13 @@ public function register()
         $this->registerBatchServices();
 
         $this->app->alias(
-            Dispatcher::class, DispatcherContract::class
+            Dispatcher::class,
+            DispatcherContract::class
         );
 
         $this->app->alias(
-            Dispatcher::class, QueueingDispatcherContract::class
+            Dispatcher::class,
+            QueueingDispatcherContract::class
```

Comment:
Can you please undo all the styling changes you did in this PR?

Reply:
Done. Sorry about that.